tests/contrib/dbapi/test_dbapi.py

```diff
@@ -30,6 +31,24 @@ def test_execute_wrapped_is_called_and_returned(self):
         assert "__result__" == traced_cursor.execute("__query__", "arg_1", kwarg1="kwarg1")
         cursor.execute.assert_called_once_with("__query__", "arg_1", kwarg1="kwarg1")
 
+    @TracerTestCase.run_in_subprocess(
+        env_overrides=dict(
+            DD_TRACE_SQL_COMMENT_INJECTION_MODE="full",
```

Comment:
should we parametrize this and test the other `DD_TRACE_SQL_COMMENT_INJECTION_MODE` configs here?

Reply:
We will add integration tests to validate each injection mode